tipsy/SettingsViewController.swift

```diff
@@ -10,41 +10,35 @@ import UIKit
 
 class SettingsViewController: UIViewController {
 
-    @IBOutlet weak var defaultTipControl: UISegmentedControl!
+    @IBOutlet private weak var defaultTipControl: UISegmentedControl!
+    
+    private let defaults = UserDefaults.standard
+    
+    // MARK: - Lifecycle Methods
+    
     override func viewDidLoad() {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        
+        if let defaultTipIndex = defaults.object(forKey: UserDefaultKeys.defaultTipIndex) as? Int {
+            defaultTipControl.selectedSegmentIndex = defaultTipIndex
+        }
+    }
+    
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
     
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destinationViewController.
-        // Pass the selected object to the new view controller.
-    }
-    */
+    // MARK: - Button Methods
+    
     @IBAction func defaultTipChosen(_ sender: Any) {
-        let defaults = UserDefaults.standard
-        defaults.set(defaultTipControl.selectedSegmentIndex, forKey: "default_tip_index")
+        defaults.set(defaultTipControl.selectedSegmentIndex, forKey: UserDefaultKeys.defaultTipIndex)
         defaults.synchronize()
     }
-    
-    override func viewWillAppear(_ animated: Bool) {
```

Comment:
I like to keep view lifecycle methods together. But this is by no means required.